### ISSUE TITLE
(Docs - Concepts) Loss Functions

### ISSUE THREAD

**Issue by guenthermi** [jina-ai/finetuner#728]
- Reference Pytorch Metric learning for the loss functions and explain the most prominent TripletMarginLoss and ArcfaceLoss (can be copied from the old documentation)
- Explain what a miner is and how to use it
